src/picosvg/svg.py

```diff
@@ -1320,11 +1333,25 @@ def _swap_elements(swaps: Iterable[Tuple[etree.Element, Sequence[etree.Element]]
                 raise ValueError("Lost parent!")
             parent.remove(old_el)
 
+    @lru_cache(maxsize=None)
```

Comment:
Isn't the unbounded cache dangerous?

Reply:
I don't think so, because I always call cache_clear() at the beginning of _update_etree()
https://github.com/googlefonts/picosvg/pull/258/files#diff-613582736f235e34aeab2643a59d44fb26bfa66e5bcba5f4c6fa195d74b0e7b9R1349

functools.lru_cache(None) is the same as functools.cache(), but the latter is only available for 3.9+
https://docs.python.org/3/library/functools.html#functools.cache